core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -181,7 +183,7 @@ func (eb *EthBroadcaster) Close() error {
 
 		close(eb.chStop)
 		eb.wg.Wait()
-
+		eb.orm.Close()
```

Comment:
If you close the ORM here, you would need to pass a context into every query so that queries are quit early when EthBroadcaster exits.

I would suggest more like the following:

```suggestion
                close(eb.chStop)
		eb.orm.Close()
		eb.wg.Wait()
```

Reply:
Awesome, thanks that makes sense

Reply:
I'll do the same for eth confirmer as well

Reply:
and eth_resender as well

Reply:
Can you confirm https://github.com/smartcontractkit/chainlink/pull/8543/files#diff-7be1b35e01071a816ae4a10044dc335896e54d0e50073a8e9f9a3e81b7e82354R74 and https://github.com/smartcontractkit/chainlink/pull/8543/files#diff-d6b76588977cc18e5b4c043aaae86416044e9b7e4233e6a79f5b05f30aa5219fR181 are correct now?